src/GraphQL/Execution/ExecutionContext.cs

```diff
@@ -28,13 +28,13 @@ public class ExecutionContext : IExecutionContext
         public Operation Operation { get; set; }
 
         /// <inheritdoc/>
-        public Fragments Fragments { get; set; } = new Fragments();
+        public Fragments Fragments { get; set; }
```

Comment:
`DocumentExecuter.BuildExecutionContext` sets `Fragments` to `document.Fragments` so this allocation was redundant.

Reply:
Can we eliminate the property’s memory use and make it just point to document.Fragments ?

Reply:
And what's the point in this property then? Let's remove it.